app/DAL/models.py

```diff
@@ -55,4 +55,16 @@ class DogImage(Base):
     createdAt = Column(DateTime, default=datetime.utcnow)
     updatedAt = Column(DateTime, onupdate=datetime.utcnow)
 
-    dog = relationship('Dog', back_populates='images')
\ No newline at end of file
+    dog = relationship('Dog', back_populates='images')
+
+
+class PossibleDogMatch(Base):
+    __tablename__ = 'possible_dog_matches'
+    id = Column(Integer, primary_key=True, autoincrement=True)
```

Comment:
@YanDav78 I don't know what you agreed on, but wouldn't it make sense to also save the details of **who** marked the dog as a possible match? Meaning, saving the current user's id in here as well.

Reply:
Nevermind, I just realized new flow. The person who searches his dog can mark a potential match only after he searched. As this stage you have the `dogId` and the `potentialMatchId` so you know exactly the person who reported the match, because it's the same one on the dogs table. meaning

```SQL
SELECT reporterId FROM dogs WHERE dogs.id = possible_dog_matches.dogId
```
Would give you the one who reported the match. Did I get that right?

Reply:
@SereneSound Nir said that when someone clicks to see the contact details of a dog it would be recorded as possible match.

Reply:
@SereneSound correct :)